src/js/component.js

```diff
@@ -125,11 +125,13 @@ class Component {
      * @type {EventTarget~Event}
      *
      * @property {boolean} [bubbles=false]
-     *           set to false so that the close event does not
+     *           set to false so that the dispose event does not
      *           bubble up
      */
     this.trigger({type: 'dispose', bubbles: false});
 
+    this.isDisposed_ = true;
```

Comment:
should we check at the top of this function and return if `isDisposed_` is true`

Reply:
That seems reasonable, triggering a dispose event if we're already disposed is going to cause issues.